backend/src/main/java/de/bund/digitalservice/ris/norms/adapter/input/restapi/controller/LogoutController.java

```diff
@@ -0,0 +1,34 @@
+package de.bund.digitalservice.ris.norms.adapter.input.restapi.controller;
+
+import jakarta.servlet.http.HttpSession;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+/**
+ * Controller for handling user logout requests.
+ * <p>
+ * This controller provides an endpoint for logging out authenticated users by invalidating their session.
+ * Upon logout, the session data stored in Redis is removed, effectively ending the user's session.
+ */
+@RestController
+@RequestMapping("/api/v1/logout")
+public class LogoutController {
```

Comment:
Do we really need this? Spring automatically creates a /logout route for this purpose